[BUG] `/user/{user_id}/service-account/{service_account_id}/group-role` route returns SQLAlchemy model, not Pydantic Schema

### Description

`/user/{user_id}/service-account/{service_account_id}/group-role` returns a straight SQLA model, not a Pydantic Schema. This causes type checking to fail. 

### Steps to Reproduce

1. Run `make types`
2. Witness errors of this ilk:
```python
across_server/routes/user/service_account/group_role/router.py:57: error: Incompatible return value type (got "across_server.db.models.ServiceAccount", expected "across_server.routes.user.service_account.schemas.ServiceAccount")  [return-value]
across_server/routes/user/service_account/group_role/router.py:89: error: Incompatible return value type (got "across_server.db.models.ServiceAccount", expected "across_server.routes.user.service_account.schemas.ServiceAccount")  [return-value]
Found 2 errors in 1 file (checked 192 source files)
```


### Expected Behavior

Should return Pydantic object, removing type error.

--- a/across_server/routes/user/service_account/group_role/router.py
+++ b/across_server/routes/user/service_account/group_role/router.py
@@ -54,8 +54,8 @@ async def assign(
     )
     group_role = await group_role_service.get(group_role_id)
 
-    service_account = await service.assign(service_account, group_role, user)
-    return schemas.ServiceAccount.model_validate(service_account)
+    role_assigned_account = await service.assign(service_account, group_role, user)
+    return schemas.ServiceAccount.model_validate(role_assigned_account)
 
 
 @router.delete(
@@ -87,5 +87,5 @@ async def remove(
     )
     group_role = await group_role_service.get(group_role_id)
 
-    service_account = await service.remove(service_account, group_role)
-    return schemas.ServiceAccount.model_validate(service_account)
+    role_removed_account = await service.remove(service_account, group_role)
+    return schemas.ServiceAccount.model_validate(role_removed_account)

